lerna-management/src/main/mima-filters/1.0.0.backwards.excludes/32-typed-support-fix-compile-error.backwards.excludes

```diff
@@ -0,0 +1,15 @@
+# typed ActorSystem 対応
+# ClassicActorSystemProvider は typed/classic 両方の ActorSystem が継承しているので compileエラーにはならない
+# ※ Akka 2.6 以上のみ対応
+
+# akka.actor.ActorSystem を使っている状況で、 Tenant を継承した別の trait 型(例: AppTenant)を使用していると Set[AppTenant] が 自動で Set[Tenant] にならず、コンパイルエラーになる
+# 型を明示することで回避可能だが、 lib 側の akka.actor.ActorSystem を受け取る apply method を削除すると楽になるため ActorSystem を受け取る apply method を削除
+# [error]      overloaded method value apply with alternatives:
+# [error]        (system: akka.actor.ClassicActorSystemProvider,tenants: Set[lerna.util.tenant.Tenant])lerna.management.stats.Metrics <and>
+# [error]        (system: akka.actor.ActorSystem,tenants: Set[lerna.util.tenant.Tenant])lerna.management.stats.Metrics
+
+
+# static method apply(akka.actor.ActorSystem,scala.collection.immutable.Set)lerna.management.stats.Metrics in interface lerna.management.stats.Metrics's type is different in current version, where it is (akka.actor.ClassicActorSystemProvider,scala.collection.immutable.Set)lerna.management.stats.Metrics instead of (akka.actor.ActorSystem,scala.collection.immutable.Set)lerna.management.stats.Metrics
+ProblemFilters.exclude[IncompatibleMethTypeProblem]("lerna.management.stats.Metrics.apply")
+# method apply(akka.actor.ActorSystem,scala.collection.immutable.Set)lerna.management.stats.Metrics in object lerna.management.stats.Metrics's type is different in current version, where it is (akka.actor.ClassicActorSystemProvider,scala.collection.immutable.Set)lerna.management.stats.Metrics instead of (akka.actor.ActorSystem,scala.collection.immutable.Set)lerna.management.stats.Metrics
+ProblemFilters.exclude[IncompatibleMethTypeProblem]("lerna.management.stats.Metrics.apply")
```

Comment:
同じ exclude が2つ書かれているように思いますが、何か理由がありますか？
`ProblemFilters.exclude[IncompatibleMethTypeProblem]("lerna.management.stats.Metrics.apply")`

Reply:
> 同じ exclude が2つ書かれている

確かに同じですね。

`+mimaReportBinaryIssues` で出るエラーメッセージから機械的に変換しているのでそのままになっています。

```
[error] lerna-management: Failed binary compatibility check against com.lerna-stack:lerna-management_2.12:1.0.0! Found 2 potential problems (filtered 22)
[error]  * static method apply(akka.actor.ActorSystem,scala.collection.immutable.Set)lerna.management.stats.Metrics in interface lerna.management.stats.Metrics's type is different in current version, where it is (akka.actor.ClassicActorSystemProvider,scala.collection.immutable.Set)lerna.management.stats.Metrics instead of (akka.actor.ActorSystem,scala.collection.immutable.Set)lerna.management.stats.Metrics
[error]    filter with: ProblemFilters.exclude[IncompatibleMethTypeProblem]("lerna.management.stats.Metrics.apply")
[error]  * method apply(akka.actor.ActorSystem,scala.collection.immutable.Set)lerna.management.stats.Metrics in object lerna.management.stats.Metrics's type is different in current version, where it is (akka.actor.ClassicActorSystemProvider,scala.collection.immutable.Set)lerna.management.stats.Metrics instead of (akka.actor.ActorSystem,scala.collection.immutable.Set)lerna.management.stats.Metrics
[error]    filter with: ProblemFilters.exclude[IncompatibleMethTypeProblem]("lerna.management.stats.Metrics.apply")
```

Reply:
ありがとうございます。なるほど。